packages/server/src/main/java/software/uncharted/terarium/hmiserver/controller/dataservice/ProjectController.java

```diff
@@ -380,7 +380,7 @@ public ResponseEntity<ProjectAsset> createAsset(
 					}
 				}
 			}
-			return ResponseEntity.status(HttpStatus.NOT_MODIFIED).build();
+			return ResponseEntity.status(HttpStatus.UNAUTHORIZED).build();
```

Comment:
I don't think this is right. This will return a 401, which our front end will then capture and redirect you to the keycloak login page.